src/include/duckdb/execution/aggregate_hashtable.hpp

```diff
@@ -31,17 +31,34 @@ struct FlushMoveState;
 
 struct aggr_ht_entry_t { // NOLINT
```

Comment:
Since this class is now used by both the aggregate and join hashtables, could you move it to a separate file and rename it to `ht_entry_t`?